journal/modernized/09-chapter-8.adoc

```diff
@@ -995,8 +995,7 @@ because it was not possible he should be holden of them:
 He is Lord of lords and King of kings;
 He can and will do great and wonderful things for those who put their trust in Him.
 Blessed, praised and magnified be his adorable name, says my soul,
-forever and forever.ling to receive counsel from the very least child,
-willing to take advice as well as give.
+forever and forever.
```

Comment:
same here.... what I deleted (top of p. 219) was actually missing on the next page (220), on second paragraph. I already added it and checked with the original book from google.